### PR TITLE
Log create exceptions

### PR DESCRIPTION
**Pull request by halkyon**

### CODE REVIEW

File: code/jobs/CreateEnvJob.php [hunk 1]
```diff
@@ -20,9 +20,12 @@ public function setUp() {
 	 */
 	public function perform() {
 		echo "[-] CreateEnvJob starting" . PHP_EOL;
+
+		$log = new DeploynautLogFile($this->args['logfile']);
```
Comment (stojg) [inline code/jobs/CreateEnvJob.php]: lol, I double checked to see if the job actually got logfile set in the creation of the job and it did. we just didn't use it?

File: code/jobs/CreateEnvJob.php [hunk 2]
```diff
@@ -20,9 +20,12 @@ public function setUp() {
...
+		$envCreate = DNCreateEnvironment::get()->byId($this->args['createID']);
+
 		// This is a bit icky, but there is no easy way of capturing a failed deploy by using the PHP Resque
```
Comment (stojg) [inline code/jobs/CreateEnvJob.php]: remove the comment, seems like copy pasta?

File: code/jobs/CreateEnvJob.php (hunk 1 above)
Comment (halkyon) [inline code/jobs/CreateEnvJob.php]: yep

File: code/jobs/CreateEnvJob.php (hunk 2 above)
Comment (halkyon) [inline code/jobs/CreateEnvJob.php]: will do

**Pull request merged**